pkgs/development/python-modules/pytest-flask/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, buildPythonPackage, fetchPypi, pytest, flask, werkzeug, setuptools_scm }:
+
+buildPythonPackage rec {
+  pname = "pytest-flask";
+  version = "0.15.0";
+
+  src = fetchPypi {
```

Comment:
Could you fetch from GitHub and enable tests https://github.com/pytest-dev/pytest-flask/tree/master/tests?

##### also add a  comment no tests in are in PyPI Tarball.

Reply:
I tried that, but failed:

```
  src = fetchFromGitHub {
    owner = "pytest-dev";
    repo = pname;
    rev = version;
    sha256 = "1yfy8bgmig83jhijbsgdyfgn4xpx45fg0qmrqaw2fadx96m3hh1v";
   };
```

the problem:

```
LookupError: setuptools-scm was unable to detect version for '/build/source'.

Make sure you're either building from a fully intact git repository or PyPI tarballs. Most other sources (such as GitHub's tarballs, a git checkout without the .git folder) don't contain the necessary metadata and will not work.

For example, if you're using pip, instead of https://github.com/user/proj/archive/master.zip use git+https://github.com/user/proj.git#egg=proj
builder for '/nix/store/9diyfam4lxdc6d2nbx0s54scwk2vs4rb-python3.7-pytest-flask-0.15.0.drv' failed with exit code 1
error: build of '/nix/store/9diyfam4lxdc6d2nbx0s54scwk2vs4rb-python3.7-pytest-flask-0.15.0.drv' failed
```

Reply:
yeah it wants a file at `pytest_flask/_version.py` with `version = '${version}'` in it.
But I feel a hack like that wouldn't be a good idea so we can leave it as is.